styles.css

```diff
@@ -20,13 +20,22 @@ h1 {
     z-index: 5;
 }
 
+h1 a {
+      color: #000000;
+      text-decoration: none;
+  }
+
+h1 a:hover {
+    color: #ABB2B9;
+}
+
 header {
-    height: 500px;
+    height: 450px;
 }
 
 nav {
     position: relative;
-    height: 350px;
+    height: 365px;
 }
 
 [class|=nav]  {
```

Comment:
what is this function? what is it doing?

Reply:
Selecting any of the classes with nav in part of the class name.